tools/nvqpp/nvq++.in

```diff
@@ -589,6 +589,13 @@ COMPILER_FLAGS="${CPPSTD} -Wno-c++20-extensions ${COMPILER_FLAGS}"
 # the resultant binary to target that specified backend.
 OBJS_TO_MERGE=""
 if [ -n "${TARGET_CONFIG}" ]; then
+	if [ "${TARGET_CONFIG}" == "nvqc" ]; then
+		LLVM_CONFIG="${LLVMBIN}llvm-config"
+		HOST_TARGET=$(${LLVM_CONFIG} --host-target)
+		if [ ${HOST_TARGET:0:6} != "x86_64" ]; then
+			error_exit "Cannot use nvqc target from non-x86_64 host at this time"
```

Comment:
[nit] I don't know the terminology used elsewhere with regards to `nvqc`, but, is `client` more suitable than `host`?

Reply:
Updated in 861fc1822